spec/features/user_interacts_with_a_horse_spec.rb

```diff
@@ -0,0 +1,30 @@
+require_relative '../spec_helper.rb'
+
+describe "when a user visits a horse page" do
+  it "they see an individual horses information" do
+    Horse.create!(name: "Frank", age: 29, total_winnings: 34000, breed_id: 1, jockey_id: 1)
```

Comment:
This isn't true association because at this point you don't have a Jockey with an id of 1 or a breed. There are many ways to make associated entities with AR, here is my preferred way for this senario:

```ruby
breed = Breed.create(name: "Breed1")
jockey = Jockey.create(name: "Jockey1")

Horse.create!(name: "Frank", age: 29, total_winnings: 3400, breed: breed, jockey: jockey)
```

ActiveRecord is smart enough to know that you are trying to add the foreign key to the Horse instance. Above is shorthand for:

```ruby
breed = Breed.create(name: "Breed1")
jockey = Jockey.create(name: "Jockey1")

Horse.create!(name: "Frank", age: 29, total_winnings: 3400, breed_id: breed.id, jockey_id: jockey.id)
```